tools/raml2html/src/Test/ReferenceTester.php

```diff
@@ -0,0 +1,476 @@
+<?php
+
+namespace EzSystems\Raml2Html\Test;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ReferenceTester
+{
+    public const TEST_REFERENCE_ROUTES = 1;
+    public const TEST_CONFIG_ROUTES = 2;
+    public const TEST_ALL_ROUTES = 3;
+
+    public const DEFAULT_FILE_LIST = [
+        'vendor/ibexa/rest/src/bundle/Resources/config/routing.yml',
+        //'vendor/ibexa/commerce-rest/src/bundle/Resources/config/routing.yaml', // Removed as of 4.4
+        // `find $dxpRoot/vendor/ibexa -name "routing_rest.y*ml"`
+        //'vendor/ibexa/admin-ui/src/bundle/Resources/config/routing_rest.yaml',
+        'vendor/ibexa/calendar/src/bundle/Resources/config/routing_rest.yaml',
+        'vendor/ibexa/cart/src/bundle/Resources/config/routing_rest.yaml',
+        'vendor/ibexa/connector-dam/src/bundle/Resources/config/routing_rest.yaml',
+        'vendor/ibexa/personalization/src/bundle/Resources/config/routing_rest.yaml',
+        'vendor/ibexa/product-catalog/src/bundle/Resources/config/routing_rest.yaml',
+        //'vendor/ibexa/scheduler/src/bundle/Resources/config/routing_rest.yaml', // prefixed /api/datebasedpublisher/v1
+        'vendor/ibexa/segmentation/src/bundle/Resources/config/routing_rest.yaml',
+        'vendor/ibexa/taxonomy/src/bundle/Resources/config/routing_rest.yaml',
+    ];
```

Comment:
This shouldn't be there but in an independent config file to ease variation between versions.

For 4.3, uncomment `'vendor/ibexa/commerce-rest/src/bundle/Resources/config/routing.yaml'`
For 3.3, use
```php
    public const DEFAULT_FILE_LIST = [
        'vendor/ezsystems/ezplatform-rest/src/bundle/Resources/config/routing.yml',
        'vendor/ezsystems/ezcommerce-rest/src/Siso/RestBundle/Resources/config/routing.yaml',
        // `find $dxpRoot/vendor/ezsystems -name "routing_rest.y*ml"`
        //'vendor/ezsystems/ezplatform-admin-ui/src/bundle/Resources/config/routing_rest.yaml',
        'vendor/ezsystems/date-based-publisher/bundle/Resources/config/routing_rest.yaml',
        'vendor/ezsystems/ezplatform-calendar/src/bundle/Resources/config/routing_rest.yaml',
        'vendor/ezsystems/ezplatform-connector-dam/src/bundle/Resources/config/routing_rest.yaml',
        'vendor/ezsystems/ezrecommendation-client/src/bundle/Resources/config/routing_rest.yaml',
    ];
```

Reply:
There will be the addition of vendor/ibexa/corporate-account/src/bundle/Resources/config/routing_rest.yaml to 4.x, see https://github.com/ibexa/corporate-account/pull/155/files#diff-cd2d6c168e688c1fe9908fbfeb46faca08e464562c38a143e38c16a6bdd1b246 and #1930